docs/concepts/overview/working-with-objects/common-labels-annotations.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Common Label and Annotation Recommendations
```

Comment:
I’d suggest changing this to “Recommended Labels and Annotations”, since they’re not in common use - but having them become common is the future intent.